vnext/.flowconfig

```diff
@@ -9,40 +9,28 @@
 ; Ideally we'd delete the base versions of files that had .windows overrides as part of the
 ; initRNLibraries build step
 <PROJECT_ROOT>/Libraries/Alert/Alert.js
-<PROJECT_ROOT>/Libraries/Animated/src/nodes/AnimatedInterpolation.js
 <PROJECT_ROOT>/Libraries/Components/Button.js
 <PROJECT_ROOT>/Libraries/Components/DatePicker/DatePicker.js
 <PROJECT_ROOT>/Libraries/Components/Keyboard/KeyboardExt.js
 <PROJECT_ROOT>/Libraries/Components/Picker/Picker.js
 <PROJECT_ROOT>/Libraries/Components/Picker/RCTPickerNativeComponent.js
 <PROJECT_ROOT>/Libraries/Components/RefreshControl/RefreshControl.js
 <PROJECT_ROOT>/Libraries/Components/SafeAreaView/SafeAreaView.js
-<PROJECT_ROOT>/Libraries/Components/ScrollView/ScrollView.js
```

Comment:
Do things being removed from this file mean you no longer have override files for them?

Reply:
If you want the "source of truth" for platform overrides you can also look at the "overrides.json"!files scattered around the repo. The stuff in the flowconfig is just where we replace something that's shared JS.